dhis-2/dhis-web/dhis-web-portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -111,6 +111,12 @@
     </init-param>
   </filter>
 
+  <filter>
+    <filter-name>org.hisp.dhis.servlet.filter.AppOverrideFilter</filter-name>
```

Comment:
@larshelge should we mention the reason for this exception to the convention in a comment so future devs don't get confused and try to change it?  Can we add comments to `web.xml`?

Reply:
That would be good :+1

Reply:
I took a look at some other places using DelegatingFilterProxy and followed their example - can use `appOverrideFilter` here if using `@Component` decorator instead of adding the Bean to `beans.xml`, so made that change [here](https://github.com/dhis2/dhis2-core/pull/5977/commits/983a3884a71eff0482cf6b41d50ab21ecd2cf0d7).  No comment needed.

Reply:
Sounds good.